### PR TITLE
Type of Facility Page

### PR DESCRIPTION
**Pull request by narin**
## Description
Choose type of facility page.  If a user is eligible for community care, they will have the option to choose between a VA medical center/clinic or a Community Care facility

## Testing done
Tested locally and with unit tests

## Screenshots
image

## Acceptance criteria
- [ ] Page contains same form elements as [mockup](https://adhoc.invisionapp.com/share/KYTTUL9GU92#/screens/382961723)

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/newAppointmentFlow.js [hunk 1]
```diff
@@ -2,13 +2,20 @@ import { getFormData } from './utils/selectors';
...
+}
+
 export default {
   home: {
     url: '/',
   },
   typeOfCare: {
     url: '/new-appointment',
-    next: 'contactInfo',
+    next: 'typeOfFacility',
```
Comment (narin) [inline src/applications/vaos/newAppointmentFlow.js]: Assuming the user is CC eligible for now

File: src/applications/vaos/sass/vaos.scss [hunk 2]
```diff
@@ -45,4 +45,11 @@
   .form-radio-buttons label {
     max-width: none;
   }
+}
+
+.vaos-form__facility-type {
```
Comment (narin) [inline src/applications/vaos/sass/vaos.scss]: Would prefer to use a utility class for this, but not sure if there's a way to add a class to a `ui:title`/legend in uiSchema.

File: src/applications/vaos/sass/vaos.scss [hunk 3]
```diff
@@ -45,4 +45,11 @@
...
+  legend {
+    margin-bottom: 30px;
```
Comment (narin) [inline src/applications/vaos/sass/vaos.scss]: Would prefer to use a utility class for this, but not sure if there's a way to add a class to a `ui:title`/`legend` in uiSchema.

File: src/applications/vaos/newAppointmentFlow.js (hunk 1 above)
Comment (jbalboni) [reply src/applications/vaos/newAppointmentFlow.js]: I think I'll write a ticket for mocking that check out. I think we know enough to do that.

File: src/applications/vaos/sass/vaos.scss (hunk 3 above)
Comment (jbalboni) [reply src/applications/vaos/sass/vaos.scss]: I think you can pass a component to the `ui:title` prop, that might get you what you need.

**Review by jbalboni — APPROVED** [review]
Looks good to me.

**Comment by narin** [comment]
Waiting for https://github.com/department-of-veterans-affairs/vets-website/pull/10757 to be merged and then will update base to be `master`.

**Review by rianfowler — APPROVED** [review]

**Pull request merged**